{{cookiecutter.project_slug}}/terraform/ec2-cluster/README.md

```diff
@@ -110,3 +110,8 @@ kubectl create secret docker-registry regcred \
 NB: AWS credentials will expire after 4 hours. If you are unable to push or pull images to ECR, you will need to reauthenticate.
 
 To simplify this, you can run `AWS_PROFILE={{cookiecutter.project_slug}} make kubecreds`
+
+# Controlplane
+
+ArgoCD and SealedSecrets are configured on the SFU controlplane.
+If you are not using the SFU controlplane you can set up your own controlplane on the EC2 instance in terraform/control_plane
```

Comment:
An external audience won't know what the control plane is. In the future, we can make a "control plane" a more official feature of the cookie cutter, where we ask users if they want to install a separate cluster for monitoring. For now, could you remove this section?